main/test/flix/Test.Exp.ArrayLit.flix

```diff
@@ -2,523 +2,523 @@ namespace Test/Exp/ArrayLit {
 
     @test
     def testArrayLit01(): Unit = region r {
-        let _ = [];
+        discard [];
```

Comment:
Does this not have an effect? In particular `r`?

Reply:
Same comment as the above, it is non-unit, so you have to discard it.